persist/sqlite/consensus_refactored_test.go

```diff
@@ -1703,3 +1713,248 @@ func TestFileContractMultipleRevisions(t *testing.T) {
 
 	n.assertContractRevisions(t, fce.ID)
 }
+
+func TestFileContractsKey(t *testing.T) {
```

Comment:
Does `TestFileContractsKey` cover everything we used to assert? It seems we're missing a couple of things like `assertContractRevisions`